solidity/contracts/bridge/Bridge.sol

```diff
@@ -1497,6 +1610,18 @@ contract Bridge is Ownable {
         return info;
     }
 
+    function fraudChallengeDepositAmount() public view returns (uint256) {
+        return fraudVerificationData.parameters.fraudChallengeDepositAmount;
+    }
+
+    function fraudChallenges(uint256 challengeKey)
+        public
+        view
+        returns (FraudVerification.FraudChallenge memory)
+    {
+        return fraudVerificationData.fraudChallenges[challengeKey];
+    }
```

Comment:
Let's make those functions `external` if we don't use them from within the contract.

Reply:
Done.